browsing.go

```diff
@@ -189,6 +189,8 @@ func (c *Client) GetInstantMix(id string, idType ItemType, limit int) ([]*Song,
 		path = "/Artists/%s/InstantMix"
 	case TypeAlbum:
 		path = "/Albums/%s/InstantMix"
+	case TypeSong:
+		path = "/Songs/%s/InstantMix"
```

Comment:
I think InstantMix is already supported in because the "/Items/%s/InstantMix" path is what should be used for songs, and I don't believe the "/Songs" path exists: https://api.jellyfin.org/#tag/InstantMix/operation/GetInstantMixFromItem

Reply:
The path "/Songs" exists https://api.jellyfin.org/#tag/InstantMix/operation/GetInstantMixFromSong. And based from the source code https://github.com/jellyfin/jellyfin/blob/b1870792b1310b58f84089c1c50deab1b7d12f24/Emby.Server.Implementations/Library/MusicManager.cs#L100, `GetInstantMixFromItem` is a more general method that dispatches the logic based on the type of the input item, while `GetInstantMixFromSong` is more specific and handles the case of generating an instant mix specifically for songs. I think that it would be more efficient to call `GetInstantMixFromSong` directly

Reply:
Ah cool - I didn't see GetInstantMixFromSong somehow!